tests/unit/test.program.js

```diff
@@ -329,15 +331,26 @@ describe('program.main', () => {
 
 
 describe('program.defaultVersionGetter', () => {
+  let root = path.join(__dirname, '..', '..');
 
-  it('returns the package version', () => {
-    let root = path.join(__dirname, '..', '..');
+  it('returns the package version in production', () => {
     let pkgFile = path.join(root, 'package.json');
     return fs.readFile(pkgFile)
-      .then((pkgData) => {
-        assert.equal(defaultVersionGetter(root),
-                     JSON.parse(pkgData).version);
-      });
+    .then((pkgData) => {
```

Comment:
Just a minor problem here: this block should be indented two spaces to show that `.then()` is a continuation of the call to `readFile()`.